service/src/test/java/uk/gov/hmcts/reform/fpl/service/DraftCMOServiceTest.java

```diff
@@ -54,6 +55,10 @@
     private CaseManagementOrder caseManagementOrder;
     private List<Element<HearingBooking>> hearingDetails;
 
+    //TODO: better way to do this? Required because of DirectionHelperService.
+    @MockBean
+    private UserDetailsService userDetailsService;
```

Comment:
Just add it to the context configuration maybe?

Reply:
does not work 😢

Reply:
Big sad